examples/systemcalls/systemcalls.c

```diff
@@ -58,31 +80,63 @@ bool do_exec(int count, ...)
  *   as second argument to the execv() command.
  *
 */
+ 
+    openlog("systemcalls.c", LOG_CONS | LOG_PID | LOG_PERROR, LOG_USER);
+
+    va_list args;
+    va_start(args, count);
+
+    char *command[count + 1];
+    for (int i = 0; i < count; i++) {
+        command[i] = va_arg(args, char *);
+    }
+    command[count] = NULL;
+   
+    fflush(NULL);
+
+    pid_t pid = fork();
+    if (pid < 0) {
+        va_end(args);
+        syslog(LOG_ERR, "do_exec: fork() failed, errno=%d (%m)", errno);
+        closelog();
+        return false;
+    }
+
+    if (pid == 0) {
+        execv(command[0], command);
+        syslog(LOG_ERR, "do_exec: execv('%s', ...) failed, errno=%d (%m)", command[0], errno);
+        _exit(127);
```

Comment:
Shouldn't need to use the `_exit` version here, just `exit` should be sufficient